lib/DxilContainer/DxilContainerAssembler.cpp

```diff
@@ -1657,9 +1657,25 @@ class DxilRDATWriter : public DxilPartWriter {
     }
 
     if (props.Node.LaunchType == NodeLaunchType::Mesh) {
+      const DxilEntrySignature &sig = entryProps.sig;
       nAttrib = {};
       nAttrib.AttribKind = (uint32_t)RDAT::NodeFuncAttribKind::MeshShaderInfo;
       RDAT::MSInfo info = {};
+      uint32_t shaderFlags = 0;
+      info.SigOutputElements = AddSigElements(sig.OutputSignature, shaderFlags);
+      funcInfo.ShaderFlags |=
+          (uint16_t)(shaderFlags &
+                     (uint16_t)DxilShaderFlags::OutputPositionPresent);
+      info.SigPrimOutputElements =
+          AddSigElements(sig.PatchConstOrPrimSignature, shaderFlags);
+      // TODO: Fill in ViewID related masks
```

Comment:
What's the story with this TODO?  Are the ViewID things the thing we decided not to implement for the preview?  If so that could be explained in this comment.

Oh, I see that it looks like this TODO comment is repeated through this function, so this may be pre-existing.  
@llvm-beanz - looks like you added those TODOs when this code was originally added.  Do you know if they're still relevant?

Reply:
Yes, I copied the code directly from AddShaderInfo, which had this comment that I'm not sure is still relevant or not.

Reply:
These were TODO for exposing regular graphics entry points in libraries for the state object API (which was part of DSL), instead of requiring them to be compiled to the native shader stage targets instead (like `vs_6_8`).  We decided not to try to expose existing graphics/compute entries in libraries because there were too many incompletes and unknowns there, and we didn't have the time/resources to get right.